sonar-db/src/main/resources/org/sonar/db/component/SnapshotMapper.xml

```diff
@@ -182,36 +158,29 @@
     order by created_at desc
   </select>
 
-
-  <update id="updateSnapshotAndChildrenLastFlagAndStatus" parameterType="map">
+  <update id="unsetIsLastFlagForComponentUuid" parameterType="map">
```

Comment:
it's not only a matter of boolean parameter. `setIsLastFlagForAnalysisUuid ` sets also the status field. It's not clear and should be renamed.